src/crawler/storybook.ts

```diff
@@ -265,6 +267,7 @@ const generateBrowserConfig = (story: Story) => {
 export const generateStorybookShotItems = (
   baseUrl: string,
   stories: Story[],
+  mask?: Mask[],
```

Comment:
Where is the actual css that masks the stuff, am I blind?

Reply:
You define here the list of CSS selectors:
https://github.com/lost-pixel/lost-pixel/blob/feature/shot-mask/src/config.ts#L173-L186